integration-test/src/main/java/com/scalar/db/api/DistributedStorageMultipleClusteringKeyScanIntegrationTestBase.java

```diff
@@ -114,6 +115,15 @@ protected int getThreadNum() {
     return THREAD_NUM;
   }
 
+  protected boolean isParallelDdlSupported() {
+    return true;
+  }
+
+  // TODO: Remove this once https://github.com/yugabyte/yugabyte-db/issues/22140 is fixed.
+  protected boolean isYugabyteDb() {
+    return false;
+  }
```

Comment:
Probably JUnit 5 would fail due to `org.junit.jupiter.engine.execution.ConditionEvaluationException: Failed to evaluate condition [org.junit.jupiter.api.condition.DisabledIfCondition]: Could not find method [isYugabyteDb] in class [$(non-JDBC-integration-test)]`?, I guess.

Reply:
How about adding the following methods to `JdbcDatabaseMultipleClusteringKeyScanIntegrationTest`?

```java
  // TODO: Remove this once https://github.com/yugabyte/yugabyte-db/issues/22140 is fixed.
  @DisabledIf("isYugabyteDb")
  @Test
  @Override
  public void scan_WithSecondClusteringKeyRange_ShouldReturnProperResult()
      throws java.util.concurrent.ExecutionException, InterruptedException {
    super.scan_WithSecondClusteringKeyRange_ShouldReturnProperResult();
  }

  // TODO: Remove this once https://github.com/yugabyte/yugabyte-db/issues/22140 is fixed.
  @DisabledIf("isYugabyteDb")
  @Test
  @Override
  public void scan_WithSecondClusteringKeyRangeWithSameValues_ShouldReturnProperResult()
      throws java.util.concurrent.ExecutionException, InterruptedException {
    super.scan_WithSecondClusteringKeyRangeWithSameValues_ShouldReturnProperResult();
  }
```

I think with this, we remove `isYugabyteDb()` things from `DistributedStorageMultipleClusteringKeyScanIntegrationTestBase`.

Reply:
Ah, I see. The override seems to work. I'm okay to take it. I'll update the PR.

Reply:
Done in https://github.com/scalar-labs/scalardb/pull/1710/commits/a981f86b58c7faf6ae2eabc3c7fb02699b8677d3.